bin/README.md

```diff
@@ -0,0 +1,18 @@
+The `exercise-gen.pl6` file must currently be run from within an exercise directory. It will retrieve data from an `example.yaml` file within that directory, and use the contained information to generate a test file using `templates/test.mustache`. If it finds a `canonical-data.json` file in `x-common` for the exercise in question it will be included.
```

Comment:
Does it matter which exercise directory?  If so, the general "from within an exercise directory" may be confusing.

If possible, it should be able to be generated while sitting in the project root (this is to say, where `.git` folder sits in the project).